### PR TITLE
YLP-2948 Updated failing tests because of the time change

### PR DESCRIPTION
**Pull request by TimCanu**
## Checklist
- [x] PR name contains the Jira ticket number
- [x] Changelog updated (Jira tickets are correctly sorted)
- [x] Code tested (unit tests and integration tests)
- [x] No console errors/warnings in integration tests
- [x] No translation files updated (except for English files)
- [x] Code formatted on every file
- [x] Copyright added on every file
- [x] End-to-end tests successful on CI

### CODE REVIEW

**Comment by TimCanu** [comment]
I purposely did not update the changelog and did not start a build on yourloops as the changes are only on test files

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 Blip is the web front end for YourLoops system.
 It is based on Tidepool Blip 1.27.
 
-## 3.3.0-beta - 2024-02-14
+## 3.4.0-beta - 2024-02-14
```
Comment (celia-rouis) [inline CHANGELOG.md:5]: **🎯 suggestion (non-blocking):** ‏Date to update

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,11 @@
...
+### Engineering use
```
Comment (celia-rouis) [inline CHANGELOG.md:6]: **🎯 suggestion (non-blocking):** ‏Also update the packages version to 3.4.0-beta? 👼 

```sh
VERSION=3.4.0-beta npm run bump-version
```

**Review by celia-rouis — APPROVED** [review]

**Pull request merged**